splunk-otel-android-volley/src/main/java/com/splunk/rum/VolleyTracingBuilder.java

```diff
@@ -41,7 +41,8 @@ public final class VolleyTracingBuilder {
     private final HttpClientAttributesExtractorBuilder<RequestWrapper, HttpResponse>
             httpClientAttributesExtractorBuilder =
                     HttpClientAttributesExtractor.builder(
-                            VolleyHttpClientAttributesGetter.INSTANCE);
+                            VolleyHttpClientAttributesGetter.INSTANCE,
+                            VolleyNetClientAttributesGetter.INSTANCE);
```

Comment:
Can you also remove the `NetClientAttributesExtractor` from lines 99-100?

Reply:
Whoops. 👍🏻